cli/commands/preview/create.ts

```diff
@@ -0,0 +1,73 @@
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import archiver from 'archiver';
+import fetch from 'node-fetch';
+import { BaseCommand, OutputFormat } from 'cli/commands/base';
+
+// This is DUMMY code for now. It's only meant as a reference for the actual implementation.
+export class PreviewCreateCommand extends BaseCommand {
+	private folder: string;
+	private archivePath: string;
```

Comment:
For me a command is a short lived function, so I would like to understand more why you think it needs "state"?

Reply:
The meaning of "short-lived" is maybe relative, but I'm not so sure that'll be the most common case. Creating/updating preview sites, sync operations, import/export are all commands that can take several minutes to complete.

Reply:
> Creating/updating preview sites, sync operations, import/export are all commands that can take several minutes to complete.

True but, does it mean it needs local state in the "class". For me a class makes sense when something external is orchestrating it. Like using it now, keeping it in state and maybe use it later on some other completely unrelated flow. 

In the case of commands, it feels like it's just a single "run" function with start and end. The functions themselves can use variables to hold state and all but I don't know what the state attached to a class is for.